cdap-ui/app/cdap/components/PipelineDetails/PipelineRuntimeArgsDropdownBtn/RuntimeArgsTabContent/ProvidedPopover.js

```diff
@@ -34,14 +35,22 @@ const mapStateToProps = (state, ownProps) => {
 const ProvidedPopover = ({containsMacros, toggleAllProvided, disabled}) => {
   if (!containsMacros || disabled) { return null; }
 
+  const target = (
+    <IconSVG
+      name="icon-caret-square-o-down"
+      onClick={(e) => {
+        if (disabled) {
```

Comment:
do we need this check? If it's `disabled`, doesn't this entire component `return null`?